duckduckgo/Android
# Dark theme for Dialog Fragment/Alert Dialogs Fixes #1411

Task/Issue URL: #1411 

### Description
Themes added for AlertDialog and RadioButton

### Steps to test this PR
1. Apply Dark Theme
2. Check for the alert dialog themes all around the app.

### UI changes
![Screenshot_20210914-132356_DuckDuckGo](https://user-images.githubusercontent.com/53831987/133219142-229e2907-f9b0-46d1-b036-2c4c7918672d.png)
![Screenshot_20210914-132401_DuckDuckGo](https://user-images.githubusercontent.com/53831987/133219151-6e8e6793-d292-4f0b-aa99-3e6d2da52ded.png)
![Screenshot_20210914-132407_DuckDuckGo](https://user-images.githubusercontent.com/53831987/133219153-0e882729-6c77-4905-a00e-195c80228a1b.png)
